agent/register.go

```diff
@@ -53,15 +56,10 @@ func Register(ctx context.Context, l logger.Logger, ac APIClient, req api.AgentR
 			}
 			return err
 		}
-		registered, resp = reg, rsp
-		return nil
-	}
 
-	// Try to register, retrying every 10 seconds for a maximum of 30 attempts (5 minutes)
-	err := roko.NewRetrier(
-		roko.WithMaxAttempts(30),
-		roko.WithStrategy(roko.Constant(10*time.Second)),
-	).DoWithContext(ctx, register)
+		registered = reg
```

Comment:
Does `registered` need to be different from `reg`?

Reply:
i don't think it does! will change